viewer/vueapp/src/components/spigraph/Spigraph.vue

```diff
@@ -135,8 +135,8 @@
             class="ml-1 records-display">
             <strong class="text-theme-accent"
               v-if="!error && recordsFiltered !== undefined">
-              Showing {{ recordsFiltered | commaString }} entries filtered from
-              {{ recordsTotal | commaString }} total entries
+              Showing {{ recordsFiltered }} entries filtered from
```

Comment:
why removing commas here and below?

Reply:
Oh I see, its on the global Vue object. Good find!